src/pages/oversikt/filtrering/arbeidsmiljøboks/ArbeidsmiljøBoks.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from 'react';
+import { Element } from 'nav-frontend-typografi';
+import Ekspanderbartpanel from 'nav-frontend-ekspanderbartpanel';
+import { Checkbox } from 'nav-frontend-skjema';
+import { Alternativ } from '../Filtrering';
+import {
+    ArbeidsmijøBehov,
+    ArbeidsmiløAlternativ,
+} from '../../../registrering/arbeidsmiljø/alternativer';
+import './arbeidsmiljøBoks.less';
+import bemHelper from '../../../../utils/bemHelper';
+
+const cls = bemHelper('arbeidsmiløBoks');
+
+interface Props {
+    tittel: string;
+    spørsmål: string;
+    alternativ: ArbeidsmiløAlternativ[];
+    valgteAlternativ: ArbeidsmijøBehov[];
+    endreValgteAlternativ: (alternativ: Alternativ, checked: boolean, type: string) => void;
+}
+
+const ArbeidsmiljøBoks: FunctionComponent<Props> = props => {
+    return (
+        <div className={cls.block}>
```

Comment:
Det er mye duplisering her, ja. Usikker på hva som er best. 
Hvis vi ikke beholder duplisering: Hvordan legger vi på informasjonsikoner på enkeltspørsmål?
Hvis vi beholder duplisering: Jeg synes i så fall at vi burde flytte inn proppene tittel, spørsmål og alternativ inn i komponentene; det gir mer mening at de vet om dette selv, i stedet for Filtrering.tsx.

Det vi _kan_ generalisere her er muligens Checkbox-komponenten.